### ISSUE TITLE
[arm32][5.1] SymbolicValue should stay small

### ISSUE THREAD

**Issue by uraimo** [uraimo/buildSwiftOnARM#54]
Saw this last month, easy fix if it's still in swift-5.1-RELEASE:

```
/buildSwiftOnARM/swift/lib/SIL/SILConstants.cpp In file included from /home/buildSwiftOnARM/swift/lib/SIL/SILConstants.cpp:13:
/home/buildSwiftOnARM/swift/include/swift/SIL/SILConstants.h:437:1: error: 
static_assert failed "SymbolicValue should stay small" static_assert(sizeof(SymbolicValue) == 2 * sizeof(void *), 
^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 1 error generated.
```

**Comment by uraimo** [comment]
See [PR27515](https://github.com/apple/swift/pull/27515).

**Comment by uraimo** [comment]
Included in 5.2.
